package.json

```diff
@@ -15,40 +15,38 @@
   "dependencies": {
     "@craftzdog/react-native-buffer": "^6.0.5",
     "@react-hook/async": "^3.1.1",
-    "@react-native-async-storage/async-storage": "^1.17.10",
+    "@react-native-async-storage/async-storage": "1.23.1",
     "@react-native-clipboard/clipboard": "^1.11.1",
     "@seald-io/react-native-scrypt": "^1.2.3",
     "@walletconnect/react-native-compat": "^2.13.0",
     "bip39": "^3.0.4",
-    "expo": "^46.0.13",
-    "expo-local-authentication": "^12.3.0",
-    "expo-modules-autolinking": "^0.10.3",
-    "expo-modules-core": "^0.11.5",
-    "expo-secure-store": "^11.3.0",
-    "expo-splash-screen": "^0.16.2",
```

Comment:
Not required, already exists as a transient dep and `expo doctor` wasn't happy with that